src/style.css

```diff
@@ -1 +1,58 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__logo {
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  object-fit: none;
```

Comment:
Why do you need all these styles? Actually, height and width are enough for the logo

Reply:
Ok, fix that.